### PR TITLE
macro code documentation formatting

### PR DESCRIPTION
**Pull request by Jeroendevr**
Related to #1754, it had one return to many

### CODE REVIEW

File: xlwings/main.py [hunk 1]
```diff
@@ -482,7 +483,7 @@ def macro(self, name):
         .. code-block:: vb.net
 
         Function MySum(arr() as integer)
-            ''' code here
+            # code here
```
Comment (fzumstein) [inline xlwings/main.py:486]: given this is VBA, `'` would make more sense?

Comment (Jeroendevr) [reply xlwings/main.py:486]: Yes I thought so, but the formatting at https://docs.xlwings.org/en/stable/api.html#xlwings.App.macro was off and I thought perhaps the ‘#’ was giving the formatting error.

Comment (fzumstein) [reply xlwings/main.py:486]: I see, no, it's the missing indentation.

**Comment by fzumstein** [comment]
replaced by 942bab2bdcd93774c603f1e88e3caacfc3c4bbec

**Pull request closed without merging**